__tests__/app.test.js

```diff
@@ -31,3 +31,20 @@ describe("Verify error handling functionality", () => {
       });
   });
 });
+describe("/api", () => {
+  test("should return an object describing all the available endpoints on our API", () => {
+    return request(app)
```

Comment:
Please see Rose's post on how to approach this endpoint in a more straightforward way that will help you later on :)